src/components/EnergySeriesChart.tsx

```diff
@@ -51,12 +51,31 @@ export const EnergySeriesChart = ({ data }: OwnProps) => {
             },
           },
         ],
-        xAxes: [{ display: false }], //Not displaying x labels to save vertical space.
+        xAxes: [
+          {
+            type: 'time',
+            time: {
+              unit: 'minute',
+              stepSize: 60,
+              displayFormats: {
+                minute: 'h:mm',
+              },
+            },
+            ticks: {
+              fontColor: data.length > 0 ? '#1C1917' : '#F3F4F6', //Hack to hide badly parsed tick values while maintaining height space for ticks. (For the sake of layout stability)
+              maxTicksLimit: 3,
+              maxRotation: 0,
+              minRotation: 0,
+              fontSize: 11,
+              lineHeight: 0.4,
+            },
+          },
+        ],
       },
       legend: { display: false },
       animation: { duration: 0 }, //To improve performance
       responsiveAnimationDuration: 0, //To improve performance
-    };
+    } as Chart.ChartConfiguration;
```

Comment:
Typescript was acting a little bit annoying when type checking the options object, but the new settings don't seem to cause any errors in chart.js, so as a quick solution, I decided to skip the type check by using a type assertion in here. A little bit of a hack, I must admit.